docs/src/components/colors.ts

```diff
@@ -0,0 +1,78 @@
+import { LitElement, css, html } from 'lit';
+import { darkTokens } from '../../theme/tokens/dark';
+
```

Comment:
![image](https://github.com/amzn/style-dictionary/assets/36734656/cd47c494-d867-4706-994e-ea41aaf0ed15)

I think the style for important marked file in FileTree component is a bit off here on dark mode